app/src/main/scala/org/alephium/explorer/persistence/DBInitializer.scala

```diff
@@ -51,10 +52,21 @@ class DBInitializer(val databaseConfig: DatabaseConfig[PostgresProfile])(
       LatestBlockSchema.table,
       HashrateSchema.table,
       TokenSupplySchema.table,
-      TransactionPerAddressSchema.table
+      TransactionPerAddressSchema.table,
+      AppStateSchema.table
     )
 
-  def createTables(): Future[Unit] = {
+  def initialize(): Future[Unit] = {
+    for {
+      _ <- createTables()
+      _ <- Migrations.migrate(databaseConfig)
```

Comment:
Does this mean the migration will run every time we start the application?

We will probably introduce a version to the system and run migration based on the version.

Reply:
no, as in migrations we can add a new column, if we want to create an index on that column we need first to create it.

Reply:
yes currently it runs every time to make it easy to use and of course we will improve the all thing in the future with a version system, I'll create a task for it.

Reply:
https://github.com/alephium/explorer-backend/issues/202